docs/chaos/index.md

```diff
@@ -0,0 +1,53 @@
+# Chaos engineering with Simmy
+
+[Simmy][simmy] is a major new companion project adding a chaos-engineering and fault-injection dimension to Polly, through the provision of policies to selectively inject faults, latency, custom behavior or fake results.
```

Comment:
We should consistently use the term "chaos engineering" or "chaos-engineering" - the former seems the correct one to me.

Reply:
There were only two places where `"chaos-engineering"` were used so, I replaced the hyphens with spaces.

Reply:
Done